game/yahtzee.cpp

```diff
@@ -0,0 +1,133 @@
+#include "yahtzee.h"
+
+Yahtzee::Yahtzee() { 
+}   
+
+void Yahtzee::startGame() {
+    std::cout << "Welcome to Yahtzee!!";
+    for (int i = 1; i <= 13; ++i)
```

Comment:
instead of precomputing the number of rounds, you should have a `checkGameComplete()` function, and just take turns until that returns `true`.